_data/experience.yml

```diff
@@ -0,0 +1,48 @@
+- company: Indian Institute of Science (IISc), Bangalore
+  url: https://iisc.ac.in/
+  title: Project Assistant, Visual Computing Lab
+  date: Jul 2021 - Current
+  location: Bengaluru, India
+  description: "Currently working at <a href=\"http://visual-computing.in/home/\" target=\"_blank\">Visual Computing Lab, Department of Computational and Data Sciences (CDS), IISc</a> and conducting research in unsupervised learning methodologies and its applications in domain adaptation (DA) under the guidance of <a href=\"https://scholar.google.co.in/citations?user=NtAsZK-2HjcC&hl=en\" target=\"_blank\">Dr. Anirban Chakraborty</a>."
```

Comment:
Can you convert this page to markdown style instead? I think that is more maintainable rather than using an HTML style ?

Reply:
I forgot how to do it.. maybe something like using markdownify?

```
{% for experience in site.data.experience %}
  <div class="experience">
    <h3>{{ experience.title }}</h3>
    <p><strong>Date:</strong> {{ experience.date }}</p>
    <p><strong>Location:</strong> {{ experience.location }}</p>
    <p>{{ experience.description | markdownify }}</p>
  </div>
{% endfor %}
```

Reply:
Some resources 

- https://stackoverflow.com/questions/15917463/embedding-markdown-in-jekyll-html
- https://jekyllrb.com/tutorials/orderofinterpretation/#:~:text=Jekyll%20converts%20Markdown%20to%20HTML,specified%20in%20the%20config%20file).